source/_integrations/yeelight.markdown

```diff
@@ -161,7 +161,10 @@ Per default, the bulb limits the number of requests per minute to 60, a limitati
 Before trying to control your light through Home Assistant, you have to set up your bulb using the Yeelight app. ( [Android](https://play.google.com/store/apps/details?id=com.yeelight.cherry&hl=fr), [IOS](https://itunes.apple.com/us/app/yeelight/id977125608?mt=8) ).
 In the bulb property, you have to enable "LAN Control" (previously called "Developer mode"). LAN Control may only be available with the latest firmware installed on your bulb.  Firmware can be updated in the application after connecting the bulb.
 Determine your bulb IP (using router, software, ping...).
-Information on how to enable "LAN Control" can be found [here](https://www.yeelight.com/faqs/lan_control).
+~~Information on how to enable "LAN Control" can be found [here](https://www.yeelight.com/faqs/lan_control).~~
```

Comment:
Could you provide a bit more context? I do not quite understand why you say there "is no official way to change LAN mode". The linked yeelight instructions seem to show how to do this.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/08460e01-b159-488e-a0da-9aa6ec4d1d79)

Reply:
https://www.reddit.com/r/homeassistant/comments/mfdlva/yeelight_no_more_lan_mode/